crabs/detection_tracking/train_model.py

```diff
@@ -88,15 +95,15 @@ def main(args) -> None:
     )
     parser.add_argument(
         "--main_dir",
-        type=str,
+        nargs="+",
         required=True,
-        help="location of images and coco annotation",
+        help="list of locations of images and coco annotations",
```

Comment:
```suggestion
        help="List of paths to directories with frames to include in the dataset.",
```
it's only the images right?

Reply:
It is the main directory of where the folder frames and annotation are